### ISSUE TITLE
Generate realistic fake data

### ISSUE THREAD

**Issue by Datamance** [lookit/lookit-api#373]
**TL;DR**: A script that generates (realistic) fake data in your local dev environment.

**Narrative**
As a developer, I am often crippled at the performance-tuning phase of development due to the dissimilarity of data volume and shape between my local environment and production/staging. Without large and realistic data sets locally, I can't accurately profile the impact of certain queries without first deploying to staging (or sometimes, even production).

This would be solved with a simple script that can populate realistic data, locally in postgres. Preferably we'd like some way to be able to _undo_ the effects of that script as well.

**Acceptance Criteria**

- [ ] A script in the root of the application that generates the necessary fake data in the local postgres environment. 
- [ ] Spot checking. Honestly we just need semi-realistic looking data in volume, and I'm not sure what kind of test suite would meaningfully cover this case.

**Implementation Notes**
Probably want to use Mimesis in tandem with django's built in test fixtures.
